docs/errors/jar-deprecated-jrubyversion.adoc

```diff
@@ -0,0 +1,62 @@
+= Deprecated version of JRuby for JRubyJar
+:page-layout: base
+:toc: right
+
+== What's going on?
+
+Starting with JRuby link:http://jruby.org/2015/05/05/jruby-1-7-20.html[1.7.20]
+there were substantial improvements introduced to JRuby core which make running
+JRuby in an "embedded" scenario more reliable. Packing a `JRubyJar` being a
+fairly typical "embedded JRuby" use-case, there is certain functionality that
+relies on these improvements.
+
+This does **not** mean older versions of JRuby won't work in `JRubyJar`
+archives but rather: you are _likely_ going to experience problems with a more
+complex use-case on an older JRuby. For example, Rails applications will not
+function properly if embedded in a `.jar` with JRuby 1.7.19.
+
```

Comment:
building a library jar is not using all those embedded features of jruby and with this it is less dependent on the jruby version. might be worth mentioning.